### DISCUSSION TITLE
IKEA outlet No converter message

### DISCUSSION THREAD

**Discussion by paulz-gh** [Koenkk/zigbee2mqtt#5659] [Devices]
I regularly see the following in my logs:
```
debug 2021-01-09 16:20:49: Received Zigbee message from 'Ventilator', type 'attributeReport', cluster 'genLevelCtrl', data '{"currentLevel":254}' from endpoint 1 wit
h groupID 0
debug 2021-01-09 16:20:49: No converter available for 'E1603/E1702' with cluster 'genLevelCtrl' and type 'attributeReport' and data '{"currentLevel":254}'
````
The IKEA outlook seems to work normally so the message does not seem to indicate any error but I would not expect this for a supported device.

I updated my 1.16.2 setup to 1.17.0 today but the message still appears..

**Comment by Koenkk** [comment]
These reports usually indicate brightness values but since this is a socket it's strange it sends these messages. Not sure what the socket is trying to tell us here. Note that this is not really an error, it just means this zigbee message will not be converted to a MQTT one.

**Reply by paulz-gh** [reply]
Hmm, brightness does indeed not seem to make much sense for an on/off switch the outlet is...
As it apparently is not something bad in my configuration I'll just ignore the messages...
